workers.py

```diff
@@ -9,6 +10,20 @@
 
 LOGGER = logging.getLogger()
 MAX_RETRIES = 3
+DEFAULT_FEATURE_LIST = [
+    'number_of_cpus',
+    'cores_per_socket',
+    'system_memory_bytes',
+    'infrastructure_type',
+    'infrastructure_vendor',
+    'bios_vendor',
+    'bios_version',
+    'bios_release_date',
+    'os_release',
+    'os_kernel_version',
+    'arch'
+]
+FEATURE_LIST = os.environ.get('FEATURE_LIST', DEFAULT_FEATURE_LIST)
```

Comment:
Here's a suggestion to make this more configurable.

How about we store this list in a json (yaml also works), and then load the json at runtime to get the Column list?

Then in an OpenShift environment, if you need to run some tests with more columns or less columns, or a completely new set of columns, all you need to do is simply exec into the `aiops-outlier-detection` pod and adjust the columns in your json per your test requirement, and then just execute your test.

The advantage is, there would be no need to make code changes or env changes or redeploy the service.
You can pretty much continue to use the same pod for trying out various columns.